src/main/java/uk/gov/register/presentation/app/PresentationApplication.java

```diff
@@ -67,11 +65,15 @@ public void run(PresentationConfiguration configuration, Environment environment
         executorService.execute(new ConsumerRunnable(configuration, updateDAO));
 
         DropwizardResourceConfig resourceConfig = environment.jersey().getResourceConfig();
-        resourceConfig.property(ServerProperties.MEDIA_TYPE_MAPPINGS, ImmutableMap.of(
+
+        ImmutableMap<String, MediaType> representations = ImmutableMap.of(
                 "csv", ExtraMediaType.TEXT_CSV_TYPE,
                 "tsv", ExtraMediaType.TEXT_TSV_TYPE,
                 "ttl", ExtraMediaType.TEXT_TTL_TYPE,
-                "json", MediaType.APPLICATION_JSON_TYPE));
+                "html", MediaType.TEXT_HTML_TYPE,
```

Comment:
to be removed (discussed IRL -- this is a separate concern from thymeleaf support)